modules/executors/main.tf

```
@@ -113,6 +113,8 @@ resource "aws_launch_template" "executor" {
       volume_type = "gp3"
       iops        = var.boot_disk_iops
       throughput  = var.boot_disk_throughput
+      encrypted   = true
```

This is 💥 if people use this code, so maybe we want to shim this with a var for "false" for awhile?

---

You mean it would depend on the `boot_disk_kms_key_id` variable being set correctly? Could we say 
```suggestion
      encrypted   = var.boot_disk_kms_key_id != null ? true : false
```
to mitigate it?

---

Wait, no that can't be it.. we set encryption to true for the docker thing as well 🤔 I'm lost, what do you mean? 😬

---

😅 `aws_launch_template`

Nevermind, new instances will just start to get encrypted. I _thought_ this part of an `aws_instance` definition, which would cause the instance to be recreated.